DataRepo/loaders/sequences_loader.py

```
@@ -194,6 +200,21 @@ def load_data(self):
         known_researchers = get_researchers()
 
         for _, row in self.df.iterrows():
+            note = self.get_row_val(row, self.headers.NOTES)
```

C. We need to get the `note` value before the `get_or_create` so we can raise an error about it.